examples/validation-app/src/__tests__/acceptance/validate.acceptance.ts

```diff
@@ -62,14 +101,14 @@ describe('validate properties', () => {
     );
   });
 
-  it('fails when creating a CoffeeShop with an invalid type', async () => {
+  it('should use sequence-level custom error handler', async () => {
     const invalidCapacityTypeCS = {
       city: 'Toronto',
       phoneNum: '416-111-1111',
       capacity: '10',
     };
     const response = await client
-      .post('/coffee-shops')
+      .patch('/coffee-shops')
```

Comment:
hmm why it changes to patch? I guess it's also related to the code change in sequence?

Reply:
In the process of creating custom error messages, the custom sequence was gobbling up the error messages generated by AJV validation and emitting its own version, and the tests had been written for this behavior. Tests using `errorMessage` would simply fail.

While technically not incorrect, it was an incomplete example of validation example. Validation and customization of errors can be done at different levels.

1. Model level: validation using `jsonSchema` and custom error messages using `errorMessage`
2. Controller-level validation (which, the example app was doing)
3. Sequence-level custom error messages (which, the example app was doing)

We cannot continue to use the custom sequence error generator as-is, since it will emit some other error in place of AJV error, and the tests will fail. However, since it shows a way to create custom errors, it makes sense to keep it - but test it in a non-disruptive manner - so let handle all errors from `PATCH` request.

Reply:
hm. Not sure if I understand the intention of changing from `POST` to `PATCH` yet. 
- If we want to show how to custom error messages using `errorMessage` and we want varieties of not just calling POST, I'm good with it, because all the tests here are POST.
- If we change it because `errorMessage` won't work with `POST`, then that would be something we need to look into.

Reply:
> If we change it because errorMessage won't work with POST, then that would be something we need to look into.

That test is especially written for the sequence modifying the error object and creating custom error object. It is not a part of AJV validation.

If we have a consensus on this test not belonging here, then we can remove it along with the custom sequence handler. I'd like to especially hear from @jannyHou.

Reply:
Had a chat with @hacksparrow, thanks for the explanation and it's fair enough to me to specify an operation to test the customized error, and let other operations return the ajv validation error as is 👍 
Probably add a comment in this test and the sequence file to explain the reason.